### PR TITLE
feat(pause): make page.pause public

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/server/supplements/injected/consoleApi.ts [hunk 1]
```diff
@@ -47,18 +59,20 @@ export class ConsoleAPI {
...
   }
 
   private _selector(element: Element) {
     if (!(element instanceof Element))
       throw new Error(`Usage: playwright.selector(element).`);
     return generateSelector(this._injectedScript, element).selector;
   }
+
+  private _resume() {
+    window.playwrightResume().catch(() => {});
```
Comment (dgozman) [inline src/server/supplements/injected/consoleApi.ts:74]: Let's rename all our bindings to `_playwrightFoo` for nicer console experience.

File: src/server/supplements/injected/consoleApi.ts [hunk 2]
```diff
@@ -17,18 +17,30 @@
 import type InjectedScript from '../../injected/injectedScript';
 import { generateSelector } from './selectorGenerator';
 
+type ConsoleAPIInterface = {
```
Comment (dgozman) [inline src/server/supplements/injected/consoleApi.ts:20]: Why empty?

File: docs/src/api/class-page.md [hunk 3]
```diff
@@ -1479,6 +1479,18 @@ The page's main frame. Page is guaranteed to have a main frame which persists du
 
 Returns the opener for popup pages and `null` for others. If the opener has been closed already the returns `null`.
 
+## async method: Page.pause
+
+Pauses script execution. Playwright will stop the script and wait for the user to either press 'Resume' button in the
```
Comment (dgozman) [inline docs/src/api/class-page.md:1484]: This sounds strange in the async world. We are not "stopping the script" - rather, the method waits for something, like a `waitForTimeout`.

Perhaps `this method will wait until user presses "Resume" button in the page overlay or calls playwright.resume() in the DevTools console` is more accurate?

**Review by dgozman — APPROVED** [review]

File: docs/src/api/class-page.md (hunk 3 above)
Comment (pavelfeldman) [reply docs/src/api/class-page.md]: I'll rephrase

**Pull request merged**